extensions/pyRevitTools.extension/pyRevit.tab/Drawing Set.panel/views.stack/Legends.pulldown/Copy Legends to Other Documents.pushbutton/script.py

```diff
@@ -96,3 +114,44 @@ def OnDuplicateTypeNamesFound(self, args):
                         'Legend already exists. Renaming to: "%s"', new_name)
                 revit.update.set_name(dest_view, new_name)
                 dest_view.Scale = src_legend.Scale
+
```

Comment:
I believe this script is becoming too long; could you please move the following lines to a function and call it here?